draft-ietf-tls-tls13.md

```diff
@@ -1036,9 +1036,10 @@ Subsequent Handshake:
 
 As the server is authenticating via a PSK, it does not send a
 Certificate or a CertificateVerify message. When a client offers resumption
-via PSK, it SHOULD also supply a "key_share" extension to the server as well
-to allow the server to decline resumption and fall back to a full handshake,
-if needed. The server responds with a "pre_shared_key" extension
+via PSK, it SHOULD also supply a "key_share" extension to the server (to provide
+forward secrecy) as well as allow the server to decline resumption and fall back
```

Comment:
This seems unduly prescriptive. It's up to people whether they want to do a DH exchange here. The reason to provide a key is to allow the full handshake.